test/integration/targets/incidental_setup_postgresql_db/tasks/main.yml

```diff
@@ -207,6 +207,9 @@
   become: yes
   shell: "echo 'SHOW SERVER_VERSION' | psql --tuples-only --no-align --dbname postgres"
   register: postgres_version_resp
+  until: postgres_version_resp is successful
```

Comment:
Was this meant to be here?

Reply:
This was done because the speedup we get from this results in postgres not having enough time to start up. If this ever merges, I think this incidental test will be removed by then.